radio/src/opentx.cpp

```diff
@@ -2897,6 +2897,7 @@ uint32_t pwrCheck()
           return e_power_on;
         }
         else if (readKeys() == (1 << KEY_ENTER)) {
+          timeForcePowerOffPressed = 0;
```

Comment:
Nah, it is wrong here. If you press Enter it switches off anyway. The 10s timeout happens you if press nothing.

Reply:
stupid me, you are right here, I did it too quickly